Sources/VideoOptionsControlBar.swift

```diff
@@ -24,13 +24,20 @@ protocol VideoOptionsControlBarDelegate: class {
     // MARK: Instance variables
     weak var delegate: VideoOptionsControlBarDelegate?
     
-    var subtitleToggled: Bool {
+    var isInFullScreen: Bool = false {
+        didSet {
+            let image: UIImage? = isInFullScreen ? UIImage(named: "iconReduce") : UIImage(named: "iconExpand")
+            toggleFullScreenButton.setImage(image, for: .normal)
+        }
+    }
+    
+    var subtitleToggled: Bool = false {
         didSet {
             selectSubtitleButton.tintColor = subtitleToggled ? .orange : .white
         }
     }
     
-    var interfaceDisabled: Bool {
+    var interfaceDisabled: Bool = false {
```

Comment:
a bool is always already false when it's initialized

Reply:
Swift complains about initialization since its a property observer